changelog/21.0/21.0.0/summary.md

```diff
@@ -50,3 +50,13 @@ $ vtctldclient --server :15999 MoveTables --target-keyspace customer --workflow
 ```
 
 Mirror rules can be inspected with `GetMirrorRules`.
+
+### <a id="new-connection-drain"/>New Connection Drain
```

Comment:
Isn't this a `New VTGate Shutdown Behavior`? It feels like we're surfacing an implementation detail to the user in a high level summary.

Reply:
Fixed via 325c619fb3